launch/navigation.launch

```diff
@@ -58,6 +58,15 @@
     <remap from="odometry/filtered" to="odometry/filtered" />
   </node>
 
+<!--Plots the rover trajectory in rviz as it drives-->
+  <node name="hectory_trajectory_server" output="screen" pkg="hector_trajectory_server" type="hector_trajectory_server" >
+    <param name="target_frame_name" type="string" value="odom" />
+    <param name="source_frame_name" type="string" value="base_link" /> 
+    <param name="trajectory_update_rate" type="double" value="4" /> 
+    <param name="trajectory_publish_rate" type="double" value="0.25" />
```

Comment:
I think we should have it publish at the same rate it updates at, 0.25 Hz is too slow since it only publishes every 4 seconds.